sov-enshrine-esdt-safe/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+#![no_std]
+
+#[allow(unused_imports)]
+use multiversx_sc::imports::*;
+use operation::EsdtSafeConfig;
+
+pub mod deposit;
+
+#[multiversx_sc::contract]
+pub trait SovEnshrineEsdtSafe:
+    deposit::DepositModule
+    + cross_chain::deposit_common::DepositCommonModule
+    + cross_chain::execute_common::ExecuteCommonModule
+    + cross_chain::storage::CrossChainStorage
+    + cross_chain::events::EventsModule
+    + utils::UtilsModule
+    + max_bridged_amount_module::MaxBridgedAmountModule
+    + multiversx_sc_modules::pause::PauseModule
+{
+    #[init]
+    fn init(
+        &self,
+        token_handler_address: ManagedAddress,
```

Comment:
I guess we don't need this contract in Sovereign, this is only part of the mvx contract on execute endpoint.